src/customMetadata/Data_Import_Field_Mapping.Payment_Elevate_API_Declined_Reason.md

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Payment Elevate API Declined Reason</label>
+    <protected>true</protected>
+    <values>
+        <field>Data_Import_Field_Mapping_Set__c</field>
+        <value xsi:type="xsd:string">Default_Field_Mapping_Set</value>
+    </values>
+    <values>
+        <field>Is_Deleted__c</field>
+        <value xsi:type="xsd:boolean">false</value>
+    </values>
+    <values>
+        <field>Required__c</field>
+        <value xsi:type="xsd:string">No</value>
+    </values>
+    <values>
+        <field>Source_Field_API_Name__c</field>
+        <value xsi:type="xsd:string">npsp__Payment_Declined_Reason__c</value>
+    </values>
+    <values>
+        <field>Target_Field_API_Name__c</field>
+        <value xsi:type="xsd:string">npsp__ Elevate_Payment_API_Declined_Reason__c</value>
```

Comment:
```suggestion
        <value xsi:type="xsd:string">npsp__Elevate_Payment_API_Declined_Reason__c</value>
```

Reply:
@callen-sfdo looks like there's an extra space here

Reply:
@Julian88Tex yep, thats a typo, thanks for the catch